src/led.h

```diff
@@ -15,4 +14,74 @@
 #define LED_h   0x12
 #define LED_dp  0x13
 
-void filldisplay(uint8_t pos, uint8_t val, __bit dp);
+const uint8_t __at (0x1000) ledtable[]
```

Comment:
I just noticed this, and did a double-take. Have you shifted the led lookup tables into the "eeprom" area beyond regular code/flash? If so, ingenious ;) - also, if so, did you already upload/populate the "eeprom" on your chip?